fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -178,7 +188,7 @@ protected ParameterTransactionDataImpl getTransactionData() {
 
     @Override
     public Resource read(String logicalId, String resourceType)
-            throws FHIRPersistenceDataAccessException, FHIRPersistenceDBConnectException {
```

Comment:
bad formatting change.  the double-indent signifies that its a continuation of the previous line so that it doesn't look like the first line of the method